mgmtd/mgmt_vty.c

```diff
@@ -570,6 +571,7 @@ void mgmt_vty_init(void)
 	 */
 	filter_cli_init();
 	route_map_cli_init();
+	affinity_map_init();
```

Comment:
It's too bad this doesn't follow the naming pattern `affinity_map_cli_init()`

Reply:
I agree, but it's used in isisd and I don't want to touch it as part of this PR. We can rename it separately later.